### PR TITLE
dev: Fixing devcontainer

### PR DESCRIPTION
**Pull request by svrooij**
I fixed the dev container, it would not start because of version mismatch (docusaurus required node >=16) and the postStart command was doing an apt install which requires sudo in latest dev containers.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,3 +23,6 @@ generate: ## Build the documentation
 
 update: ## Pull main from hacs/documentation
 	git pull upstream main;
+
+upstream: ## Add upstream to be able to update
+    git remote add upstream https://github.com/hacs/documentation.git;
```
Comment (ludeeus) [inline Makefile:28]:
```suggestion
```
Not needed

File: Makefile [hunk 2]
```diff
@@ -11,8 +11,8 @@ start: generate ## Start the documentation server
 	yarn start;
 
 bootstrap: ## Run yarn
-	apt update
-	apt install -y python3-pip
+	sudo apt update
+	sudo apt install -y python3-pip
```
Comment (ludeeus) [inline Makefile:15]:
```suggestion
```
Not needed

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "HACS Documentation",
-	"image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:0-14",
+	"image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:16",
```
Comment (ludeeus) [inline .devcontainer/devcontainer.json:3]: 18 is now LTS, why do 16 instead of 18?

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 3 above)
Comment (svrooij) [reply .devcontainer/devcontainer.json:3]: Will fix your requested changes tomorrow.

**Review by ludeeus — APPROVED** [review]

**Pull request merged**